### PR TITLE
Updated content on OpenSearch plugin availability

### PR DESCRIPTION
**Pull request by diawarr**
- Updated content and plugin list now that the inclusion of new OpenSearch plugins has been deployed to production

### CODE REVIEW

File: docs/products/opensearch/reference/plugins.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Plugins available with Aiven for OpenSearch
 ===========================================
 
-Aiven for OpenSearch includes by default a standard a set of plugins. We are also working on adding a further set of new OpenSearch plugins. The exact plugins are listed in the sections further down this page.
+Aiven for OpenSearch includes a standard a set of plugins. In addition to the plugins that were previously available in Aiven for Elasticsearch, the standard set of plugins also includes some that are new for OpenSearch.
```
Comment (anelook) [inline docs/products/opensearch/reference/plugins.rst:4]: a standard set (extra 'a')

Comment (anelook) [inline docs/products/opensearch/reference/plugins.rst:4]: I'm not sure I understand correctly this phrase - "includes some that are new for OpenSearch."  The plugins  that were added recently are not new for OpenSearch. I'd rephrase it - something like this - 
"the list now includes additional plugins designed and developed specifically for OpenSearch".

File: docs/products/opensearch/reference/plugins.rst [hunk 2]
```diff
@@ -21,17 +21,13 @@ These plugins are enabled on all Aiven for OpenSearch services by default:
 * Mapper Size
```
Comment (anelook) [inline docs/products/opensearch/reference/plugins.rst:21]: Some plugins have links, others don't, why so and what's the difference?

**Review by anelook — CHANGES_REQUESTED** [review]
Thx for the article updates :) 
I've added couple of notes. There is a typo and I suggest some rephrasing.

(optional idea) I also wonder if the list of plugins can be divided into two or more subgroups to increase readability, I assume that those that have links are different from those that don't.

File: docs/products/opensearch/reference/plugins.rst (hunk 1 above)
Comment (diawarr) [reply docs/products/opensearch/reference/plugins.rst]: True, well spotted - that's now fixed ;)

Comment (diawarr) [reply docs/products/opensearch/reference/plugins.rst]: Sure, that's now edited ;)

File: docs/products/opensearch/reference/plugins.rst (hunk 2 above)
Comment (diawarr) [reply docs/products/opensearch/reference/plugins.rst]: The ones without links didn't have any previously either, but I've added them in now ;)

**Review by anelook — APPROVED** [review]
Thank you, looks great!
